<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(54, 149, 231, 255), stop:1 rgba(255, 255, 255, 255))
}</string>
  </property>
  <widget class="QLabel" name="QuestionLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>299</width>
     <height>78</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
font: 25 12pt &quot;Calibri&quot;;
background-color: &quot;lightgrey&quot;
}</string>
   </property>
   <property name="text">
    <string>This is where the question text will be.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="submit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
font: 25 12pt &quot;Calibri&quot;;
background-color: &quot;lightgrey&quot;
}</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="answerBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
font: 25 12pt &quot;Calibri&quot;;
background-color: &quot;lightgrey&quot;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Type your answer here</string>
   </property>
  </widget>
  <widget class="QToolButton" name="hintButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
font: 25 12pt &quot;Calibri&quot;;
background-color: &quot;lightgrey&quot;
}</string>
   </property>
   <property name="text">
    <string>Hint</string>
   </property>
  </widget>
  <widget class="QLabel" name="hintLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
font: 25 12pt &quot;Calibri&quot;;
color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="emptyans">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
color:&quot;red&quot;;
background-color:rgba(255, 255, 255, 0)
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
